### PR TITLE
BLT-2512: adding execute in VM for config.

### LINKED ISSUE #2512: Config Import Fails
My system information:
* Operating system type: Mac OS
* Operating system version: 10.11.6
* BLT version: 9 Beta

Output of `blt doctor`:
```
blt doctor
Drupal VM was detected. Running blt doctor inside of VM...
Executing command cd /var/www/<project>/docroot/.. && ./vendor/bin/blt doctor inside of Drupal VM...
[error]  Could not find VM. Please ensure that the VM machine name matches project.machine_name 
+---------------------------+------------------------------------------------------+
| Property                  | Value                                                |
+---------------------------+------------------------------------------------------+
| %paths.%root              | /var/www/<project>/docroot                               |
| %paths.%site              | sites/default                                        |
| %paths.%modules           | sites/all/modules                                    |
| %paths.%themes            | sites/all/themes                                     |
| %paths.%config-sync       | /var/www/<project>/config/default                        |
| %paths.%files             | sites/default/files                                  |
| %paths.%temp              | /tmp                                                 |
| %paths.%private           | /var/www/<project>/files-private                         |
| admin-theme               | seven                                                |
| alias-searchpaths.0       | /var/www/<project>/drush/sites                           |
| blt-version               | 9.0.0-beta2                                          |
| bootstrap                 | Successful                                           |
| composer-version          | Composer version 1.6.3 2018-01-31 16:28:17           |
| config-sync               | /var/www/<project>/config/default                        |
| db-driver                 | mysql                                                |
| db-hostname               | localhost                                            |
| db-name                   | drupal                                               |
| db-password               | drupal                                               |
| db-port                   | 3306                                                 |
| db-status                 | Connected                                            |
| db-username               | drupal                                               |
| drupal-settings-file      | sites/default/settings.php                           |
| drupal-version            | 8.4.4                                                |
| drush-alias-files.0       | /var/www/<project>/drush/sites/blted8.site.yml           |
| drush-alias-files.1       | /var/www/<project>/drush/sites/<project>.site.yml            |
| drush-conf.0              | /var/www/<project>/vendor/drush/drush/drush.yml          |
| drush-conf.1              | /var/www/<project>/drush/drush.yml                       |
| drush-conf.2              | /var/www/<project>/docroot/sites/default/local.drush.yml |
| drush-script              | /var/www/<project>/vendor/bin/drush                      |
| drush-temp                | /tmp                                                 |
| drush-version             | 9.1-dev-g87d3c36                                     |
| files                     | sites/default/files                                  |
| install-profile           | <project>                                                |
| modules                   | sites/all/modules                                    |
| php-bin                   | /usr/bin/php7.1                                      |
| php-conf.0                | /etc/php/7.1/cli/php.ini                             |
| php-os                    | Linux                                                |
| private                   | /var/www/<project>/files-private                         |
| root                      | /var/www/<project>/docroot                               |
| site                      | sites/default                                        |
| stacks.drupal-vm.inited   | false                                                |
| stacks.dev-desktop.inited | false                                                |
| temp                      | /tmp                                                 |
| theme                     | today                                                |
| themes                    | sites/all/themes                                     |
| uri                       | http://local.blted8.com                              |
+---------------------------+------------------------------------------------------+
+---------------------------------------+--------------------------------------------------------------+
| Check                                 | Problem                                                      |
+---------------------------------------+--------------------------------------------------------------+
| BehatCheck:checkBaseUrl:uri           | base_url in tests/behat/local.yml does not match the site    |
|                                       | URI.                                                         |
|                                       |   Behat base_url is set to                                   |
|                                       | http://local.<project>.com.                                      |
|                                       |   Drush site URI is set to                                   |
|                                       | http://local.blted8.com.                                     |
| NodeCheck:checkNodeVersionFileExists  | Neither .nvmrc nor .node-version file found in repo root.    |
| SimpleSamlPhpCheck:checkSimpleSamlPhp | /var/www/<project>/vendor/simplesamlphp/simplesamlphp/metadata/s |
|                                       | aml20-idp-remote.php is missing. Run `blt                    |
|                                       | source:build:simplesamlphp-config`.                          |
| WebUriCheck:checkUriResponse          | Did not get a response from http://local.blted8.com          |
|                                       | Is your *AMP stack running?                                  |
|                                       | Is your web server configured to serve this URI from         |
|                                       | /var/www/<project>/docroot?                                      |
|                                       | Is options.uri set correctly in                              |
|                                       | /var/www/<project>/docroot/sites/default/local.drush.yml?        |
+---------------------------------------+--------------------------------------------------------------+
[error]  BLT Doctor discovered one or more critical issues. 
Connection to 127.0.0.1 closed.
+---------------------------+---------------------------------------------+
| Property                  | Value                                       |
+---------------------------+---------------------------------------------+
| %paths.%root              | /var/www/<project>/docroot                      |
| %paths.%site              | sites/default                               |
| %paths.%modules           | sites/all/modules                           |
| %paths.%themes            | sites/all/themes                            |
| %paths.%config-sync       | /var/www/<project>/config/default               |
| %paths.%files             | sites/default/files                         |
| %paths.%temp              | /tmp                                        |
| %paths.%private           | /var/www/<project>/files-private                |
| admin-theme               | seven                                       |
| alias-searchpaths.0       | /var/www/<project>/drush/sites                  |
| blt-version               | 9.0.0-beta2                                 |
| bootstrap                 | Successful                                  |
| composer-version          | Composer version 1.5.2 2017-09-11           |
|                           | 16:59:25                                    |
| config-sync               | /var/www/<project>/config/default               |
| db-driver                 | mysql                                       |
| db-hostname               | localhost                                   |
| db-name                   | drupal                                      |
| db-password               | drupal                                      |
| db-port                   | 3306                                        |
| db-status                 | Connected                                   |
| db-username               | drupal                                      |
| drupal-settings-file      | sites/default/settings.php                  |
| drupal-version            | 8.4.4                                       |
| drush-alias-files.0       | /var/www/<project>/drush/sites/blted8.site.yml  |
| drush-alias-files.1       | /var/www/<project>/drush/sites/<project>.site.yml   |
| drush-conf.0              | /var/www/<project>/vendor/drush/drush/drush.yml |
| drush-conf.1              | /var/www/<project>/drush/drush.yml              |
| drush-script              | /usr/local/bin/drush                        |
| drush-temp                | /tmp                                        |
| drush-version             | 9.1-dev-g87d3c36                            |
| files                     | sites/default/files                         |
| install-profile           | <project>                                       |
| modules                   | sites/all/modules                           |
| php-bin                   | /usr/bin/php7.1                             |
| php-conf.0                | /etc/php/7.1/cli/php.ini                    |
| php-os                    | Linux                                       |
| private                   | /var/www/<project>/files-private                |
| root                      | /var/www/<project>/docroot                      |
| site                      | sites/default                               |
| stacks.drupal-vm.inited   | true                                        |
| stacks.dev-desktop.inited | false                                       |
| temp                      | /tmp                                        |
| theme                     | today                                       |
| themes                    | sites/all/themes                            |
| uri                       | http://local.<project>.com                      |
+---------------------------+---------------------------------------------+
PHP Notice:  Undefined index: php-mysql in <project>/vendor/acquia/blt/src/Robo/Commands/Doctor/DbCheck.php on line 85

Notice: Undefined index: php-mysql in <project>/vendor/acquia/blt/src/Robo/Commands/Doctor/DbCheck.php on line 85
+------------------------------------------+--------------------------------------------------------------+
| Check                                    | Problem                                                      |
+------------------------------------------+--------------------------------------------------------------+
| DbCheck:checkDbConnection                | Could not connect to MySQL database.                         |
|                                          |                                                              |
|                                          | Is your *AMP stack running?                                  |
|                                          | Are your database credentials correct?                       |
|                                          |   db-driver: mysql                                           |
|                                          |   db-hostname: localhost                                     |
|                                          |   db-username: drupal                                        |
|                                          |   db-password: drupal                                        |
|                                          |   db-name: drupal                                            |
|                                          |   db-port: 3306                                              |
|                                          |                                                              |
|                                          | To verify your mysql credentials, run `mysql -u drupal -h    |
|                                          | localhost -pdrupal -P 3306`                                  |
|                                          |                                                              |
|                                          | Are you using the correct PHP binary?                        |
|                                          | Is PHP using the correct MySQL socket?                       |
|                                          |   php-os: Linux                                              |
|                                          |   php-bin: /usr/bin/php7.1                                   |
|                                          |   php-conf: /etc/php/7.1/cli/php.ini                         |
|                                          |   php-mysql:                                                 |
|                                          |                                                              |
|                                          | Are you using the correct site and settings.php file?        |
|                                          |   site: sites/default                                        |
|                                          |   drupal-settings-file: sites/default/settings.php           |
|                                          |                                                              |
|                                          | To verify, run `drush sqlc`                                  |
|                                          |                                                              |
| DrupalCheck:checkCoreExists              | Drupal core is missing!                                      |
|                                          |                                                              |
|                                          |   Looked for docroot in /var/www/<project>/docroot.              |
|                                          | Check and re-install your composer dependencies.             |
| DrupalCheck:checkDrupalInstalled         | Drupal is not installed.                                     |
|                                          |                                                              |
|                                          | Run `blt drupal:install` to install Drupal locally.          |
| DrupalVmCheck:checkDrupalVm:alias        | The drush alias assigned to drush.aliases.local does not     |
|                                          | exist in your drush aliases file.                            |
|                                          |   drush.aliases.local is set to @@<project>.local                |
| FileSystemCheck:checkFileSystem:%private | Private files directory does not exist.                      |
|                                          |                                                              |
|                                          | Create /var/www/<project>/files-private.                         |
|                                          | Installing Drupal will create this directory for you.        |
|                                          | Run `blt drupal:install` to install Drupal, or run `blt      |
|                                          | setup` to run the entire setup process.                      |
|                                          | Otherwise, run `mkdir -p /var/www/<project>/files-private`.      |
|                                          |                                                              |
| NodeCheck:checkNodeVersionFileExists     | Neither .nvmrc nor .node-version file found in repo root.    |
| PhpCheck:checkPhpDateTimezone            | PHP setting for date.timezone is not set.                    |
|                                          |   Define date.timezone in /usr/local/etc/php/7.1/php.ini     |
| SettingsFilesCheck:local-settings        | Could not find local settings file.                          |
|                                          | Your local settings file should exist at                     |
|                                          | /var/www/<project>/docroot/sites/default/settings/local.settings |
|                                          | .php.                                                        |
| SimpleSamlPhpCheck:checkSimpleSamlPhp    | <project>/vendor/simplesamlphp/simplesam |
|                                          | lphp/metadata/saml20-idp-remote.php is missing. Run `blt     |
|                                          | source:build:simplesamlphp-config`.                          |
+------------------------------------------+--------------------------------------------------------------+
[error]  BLT Doctor discovered one or more critical issues. 
```

When I run this command:
```
blt sync
```

I get the following output:
```
> drupal:config:import
[Acquia\Blt\Robo\Tasks\DrushTask] Running /Users/mike.madison/git/mtatd/vendor/bin/drush @mtatd.local cc drush --no-interaction --ansi in /Users/mike.madison/git/mtatd/docroot
 [success] 'drush' cache was cleared.
 [error]  The command could not be executed successfully (returned:  [success] 'drush' cache was cleared.
, code: 0) 
[Acquia\Blt\Robo\Tasks\DrushTask]  Exit code 1  Time 1.242s
[error]  Failed to import configuration! 
[error]  Command `drupal:config:import ` exited with code 1. 
[error]  Command `drupal:update ` exited with code 1. 
```

And I expected this to happen:
It would appear that the config import command is not executing inside the VM and is failing as a result. Adding `* @executeInDrupalVm` to src/Robo/Commands/Setup/ConfigCommand.php resolved the problem for me.

Note that BLT doctor makes it look like I don't have a VM... but I definitely do and it's working (mostly).

### PR DESCRIPTION
**Pull request by mikemadison13**
Fixes #2512  .

Changes proposed:
- running the config command inside the VM

### CODE REVIEW

**Comment by grasmash** [comment]
FYI, it you write your commit messages like "Fixes #2512 ..." GitHub will associate the PR with the issue and close the issue when the PR is merged.

**Comment by mikemadison13** [comment]
@grasmash no, i ran into a similar issue with the config import step in BLT 9.x. This is for #2512

File: src/Robo/Commands/Setup/ConfigCommand.php [hunk 1]
```diff
@@ -26,6 +26,8 @@ public function update() {
    * @command drupal:config:import
    * @aliases dci setup:config-import
    *
+   * @executeInDrupalVm
+   *
```
Comment (grasmash) [inline src/Robo/Commands/Setup/ConfigCommand.php]: Should we also add this to `drupal:update`?

Comment (mikemadison13) [reply src/Robo/Commands/Setup/ConfigCommand.php]: @grasmash i'm not sure, adding the above fixed sync for me. i'm not sure if it is needed elsewhere or not?

**Pull request merged**